CHANGELOG.md

```diff
@@ -0,0 +1,26 @@
+# Changelog
+
+## 0.9.0 (2024-09-20)
+
```

Comment:
Unless there's a more idiomatic way to do this (via a PR?):

After we're done merging everything, let's push a change to this PR's branch to include a blurb about this being the first release of finch-daemon.

Something like:

"This is the first release of Finch Daemon..."